packages/vue/src/components/molecules/SfRadio/SfRadio.vue

```diff
@@ -14,7 +14,7 @@
         :value="value"
         :checked="isChecked"
         :disabled="disabled"
-        @input="inputHandler"
+        @change="changeHandler"
```

Comment:
```
@change="changeHandler"
@input="inputHandler"
```

Reply:
Ok I finally changed it :) Sorry for misunderstanding.

Reply:
> Hi there, thank for the fix. Do you think it will be possible to get it in the next release?
> We still have plenty of users with older browser who experience the bug.
> Thanks

@mayashavin can we include it in earlier release as it is important for older browsers users?